redaxo/src/addons/cronjob/lib/manager.php

```diff
@@ -109,7 +109,7 @@ public function log($success, $message)
             ($success ? 'SUCCESS' : 'ERROR'),
             ($this->id ?: '--'),
             $name,
-            strip_tags($message),
+            str_replace("\n", '\n', strip_tags($message)),
```

Comment:
sollten wir diese "umwandlung" lieber direkt im rex_log_file machen, damit es für alle callsites gilt?

Reply:
.. wäre vielleicht besser, ja - dürfte ja generell nicht schaden...

Reply:
Müssen wir noch beachten, wenn `\n` als String bereits in der Logmessage vorkommt? (Zum Beispiel durch `\\n` maskieren, muss dann aber bei der Rückumwandlung auch berücksichtigt werden..)
Oder ein Edge case den wir ignorieren?

Reply:
Kann man glaub ich ignorieren... bzw - würde das dann automatisch zum \\n, oder? In der Klasse rex_log_entry müsste man dann die Funktion __toString() abändern - str_replace(["\r", "\n"], '', $data) zu str_replace(["\r", "\n"], ['','\n'], $data)  - oder? Und bei der getData() das Array durchgehen und wieder zurück-ersetzen... müsste gehen, oder?

Reply:
So dann: https://github.com/redaxo/redaxo/pull/1542 - damit wäre dieser hier hinfällig...

Reply:
> Oder ein Edge case den wir ignorieren?

erstmal ignorieren 👍